WordPress/src/main/res/values/strings.xml

```diff
@@ -4336,4 +4336,6 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="jetpack_feature_card_menu_item_remind_me_later">Remind me later</string>
     <string name="jetpack_feature_card_menu_item_hide_this">Hide this</string>
     <string name="gutenberg_native_featured" tools:ignore="UnusedResources" a8c-src-lib="gutenberg">Featured</string>
+
+    <string name="install_play_store_to_get_jetpack">Please install Google Play Store to get the Jetpack app</string>
```

Comment:
Worth noting that this new string won't be sent to translation until next code freeze happens so won't be translated to end users until 21.6

Reply:
Thank you for making this clear @AliSoftware 👍 

I think having the crash fix on this release would be good enough even with just the English text :)